corehq/apps/app_manager/suite_xml.py

```diff
@@ -1099,7 +1099,7 @@ def _case_tile_template_string(self):
         with open(os.path.join(
                 os.path.dirname(__file__), "case_tile_templates", "tdh.txt"
         )) as f:
-            return f.read()
+            return unicode(f.read())
```

Comment:
> Is decode("utf-8") a better solution to that issue? If so, I'll change it over.

Yes. `unicode(f.read())` will use the default encoding (ASCII), so if that file ever gains a non-ASCII character it will fail. `f.read().decode("utf-8")` is the right way to do it (assuming the encoding of that file is actually UTF-8).

Another equally correct way to do it would be `unicode(f.read(), "utf-8")`